docs/changelog.txt

```diff
@@ -54,22 +54,43 @@ Template for new versions:
 ## New Tools
 
 ## New Features
+- ``cuboid``: construct from ``df::map_block*``, ``forBlock`` iterator to access map blocks in cuboid
+- ``cuboid``: ``clamp(cuboid other)``, ``clampNew(cuboid other)`` for cuboid intersection. ``clampNew`` returns new cuboid instead of modifying.
+- ``Units::isUnitInBox``, ``getUnitsInBox``: add versions that take cuboid, filter fn parameter for ``getUnitsInBox``
+- ``Units::getProfession``: get a unit's profession type accounting for fake identity
+- ``Units::getCasteRaw``: get a caste_raw from a unit or race and caste
```

Comment:
These are not player-visible so are not "New Features" as we defined them. I think they are better off in the `API` section